omero/developers/Matlab.rst

```diff
@@ -741,15 +741,15 @@ image ``image_id``::
 
 For existing file annotations, it is possible to replace the content of the
 original file without having to recreate a new file annotation using the
-:source:`updateFileAnnotation <components/tools/OmeroM/src/annotations/updateFileAnnotation.m>` function.
+:matlab_source:`updateFileAnnotation <components/tools/OmeroM/src/annotations/updateFileAnnotation.m>` function.
 If the file annotation has been retrieved from the server as
 ``fileAnnotation``, then the content of its ``OriginalFile`` can be replaced
 by the content of ``local_file_path`` using::
 
     updateFileAnnotation(session, fileAnnotation, local_file_path);
 
 .. seealso::
-  :source:`WriteData.m <examples/Training/matlab/WriteData.m>`
+  :matlab_source:`WriteData.m <examples/Training/matlab/WriteData.m>`
```

Comment:
I didn't extract these (though maybe I should have)

Reply:
Certainly could see the examples going to a different repo.